assets/prototype/domain/eventEditor/ticketsList/ticketPriceCalculator/TicketPriceTotalRow.js

```diff
@@ -0,0 +1,62 @@
+import {Field} from 'react-final-form';
+import {Button} from '@blueprintjs/core';
+// just temporary
+import styles from './inlineStyles';
+
+// need to change these based on site i18n config
+const currencySign = '$';
+const currencySignB4 = true;
+const decimalPlaces = 2;
+// const decimalMark = '.';
+// const thousandsSep = ',';
+
+const  formatPriceAmount = (amount) => amount === undefined ? '' : Number.parseFloat(amount).toFixed(decimalPlaces);
```

Comment:
> Should we check also if that string is a number?

which string? `amount` ? It's being parsed to a float

> We might extract something into some utils later.

oh ya this will absolutely go into some kind of utility file somewhere